### PR TITLE
Implement Alluxio ui charts

### PR DESCRIPTION
**Pull request by WilliamZapata**
Vendor fonts and replace any themes' choices with our selection.
Bump some dependencies, re-shrinkwrap.
Add a line graph control, use it to consume given data, truncate and restructure data on receipt.

screen shot 2019-02-25 at 5 17 08 pm

### CODE REVIEW

File: alluxio-ui/master/src/store/metrics/reducer.tsx [hunk 1]
```diff
@@ -47,6 +50,17 @@ export const metricsReducer: Reducer<IMetricsState> = (state = initialMetricsSta
     case MetricsActionTypes.FETCH_REQUEST:
       return {...state, loading: true};
     case MetricsActionTypes.FETCH_SUCCESS:
+      const timeSeriesMetrics: LineSerieData[] = [];
+      action.payload.data.timeSeriesMetrics.map((item: any) => {
+        // only push the latest 20 points of data
+        timeSeriesMetrics.push({
+          id: item.name,
+          xAxisLabel: 'Time Stamp',
+          yAxisLabel: 'Percent (%)',
```
Comment (calvinjia) [inline alluxio-ui/master/src/store/metrics/reducer.tsx]: The y axis might not always be percentage

Comment (WilliamZapata) [reply alluxio-ui/master/src/store/metrics/reducer.tsx]: Good point, would you like me to keep a map of name to x/y labels? as long as the name does not change, it should work.

Comment (calvinjia) [reply alluxio-ui/master/src/store/metrics/reducer.tsx]: For now let's leave it as is. I think it would be better to do on the server side.

Comment (WilliamZapata) [reply alluxio-ui/master/src/store/metrics/reducer.tsx]: Makes sense

**Comment by calvinjia** [comment]
@WilliamZapata Would it be possible to only graph the first quadrant, since we know that the numbers will always be positive and let the first X timestamp act as `0` on the axis?

File: alluxio-ui/master/src/containers/pages/Metrics/Metrics.tsx [hunk 2]
```diff
@@ -299,6 +301,15 @@ export class Metrics extends React.Component<AllProps> {
...
             ))}
+            <div className="col-12">
+              <div className="row">
+                {data.timeSeriesMetrics.map((metric: LineSerieData) => (
+                  metric.data.length && <LineGraph key={metric.id} data={[metric]} xAxisLabel={metric.xAxisLabel}
+                                                   xAxisUnits={metric.xAxisUnits} yAxisLabel={metric.yAxisLabel}
+                                                   yAxisUnits={metric.yAxisUnits}/>
+                ))}
+              </div>
+            </div>
```
Comment (calvinjia) [inline alluxio-ui/master/src/containers/pages/Metrics/Metrics.tsx]: The two graphs we currently have can replace the `Master Gauges` section at the top of this page.

**Comment by calvinjia** [comment]
@WilliamZapata Thanks for the update. For the Alluxio % space used, could we only display the first quadrant? Currently the zero shows up in the middle as if the percentage could go negative.

**Comment by WilliamZapata** [comment]
@calvinjia Good point, I fixed the scale between 0 and 100.

**Pull request merged**